### PR TITLE
Create all community informations files

### PR DESCRIPTION
**Pull request by shiipou**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -20,19 +20,14 @@ Steps to reproduce the behavior:
 **Expected behavior**
 A clear and concise description of what you expected to happen.
 
-**Screenshots**
+**Screenshots and/or logs**
 If applicable, add screenshots to help explain your problem.
```
Comment (alexesprit) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Could be also changed to:
> If applicable, add screenshots and/or logs to help explain your problem.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
```
Comment (alexesprit) [inline CODE_OF_CONDUCT.md]: I suggest to keep consistent style of lists, e.g. where a dash symbol is used, as it's used in most files.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (shiipou) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks, I'll change it.

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+name: Question
+about: Ask us help in something
+title: ''
+labels: question
+assignees: ''
+
+---
+
+**Describe your aim**
+A clear and concise description of what you want
```
Comment (alexesprit) [inline .github/ISSUE_TEMPLATE/question.md]: Missing dot in the end of sentence.

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+**Ask your question**
+I can't compile the project, can you help me ?
```
Comment (alexesprit) [inline .github/ISSUE_TEMPLATE/question.md]: Redundant space before question mark.

**Pull request merged**